libbeat/docs/shared-faq.asciidoc

```diff
@@ -138,7 +138,7 @@ This is not a SSL problem. Make sure that {ls} is running and that there is no f
 
 A firewall is refusing the connection. Check if a firewall is blocking the traffic on the client, the network, or the
 destination host.
-endif::only-elasticsearch[]
+endif::no_logstash_output[]
```

Comment:
For consistency?
```suggestion
endif::[]
```

Reply:
Although, I do see the benefit of naming all of the `endif`s. It does make it easier to see which if statement it is ending...

Reply:
yah there are a few places where it clarifies rather than clutters the coding.